hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanActionExecutor.java

```diff
@@ -73,7 +73,8 @@ HoodieCleanerPlan requestClean(HoodieEngineContext context) {
       CleanPlanner<T, I, K, O> planner = new CleanPlanner<>(context, table, config);
       Option<HoodieInstant> earliestInstant = planner.getEarliestCommitToRetain();
       context.setJobStatus(this.getClass().getSimpleName(), "Obtaining list of partitions to be cleaned");
-      List<String> partitionsToClean = planner.getPartitionPathsToClean(earliestInstant);
+      boolean isDropPartition = planner.isDropPartition(earliestInstant);
```

Comment:
guess we have to rethink this a bit. We have diff clean policies and I am afraid this may not work for other policies. for eg, default is KEEP_LATEST_COMMITS and it might work, but for KEEP_LATEST_FILE_VERSIONS, earliestInstant is empty.